src/schema.ts

```diff
@@ -1,7 +1,7 @@
 import {Type} from 'vega-lite/src/type';
 import {Channel} from 'vega-lite/src/channel';
 import {autoMaxBins} from 'vega-lite/src/bin';
-import {TimeUnit, TIMEUNITS} from 'vega-lite/src/timeunit';
+import {TimeUnit, TIMEUNITS, convertDate} from 'vega-lite/src/timeunit';
```

Comment:
I don't see a PR for convertDate in Vega-Lite yet.  
Maybe name it `applyTimeUnit`? 

(Ideally I want it to be `vl.timeUnit.apply` but `apply` is a native JS method.  Any better verb that works?  Maybe just `convert` so we get `vl.timeUnit.convert`? )